src/main/java/frc/robot/subsystems/Shooter/Shooter.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.subsystems.Shooter;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Shooter extends SubsystemBase {
+    private TalonFX master = new TalonFX(Ports.Shooter.MASTER);
+    private VictorSPX slave1 = new VictorSPX(Ports.Shooter.SLAVE1);
+    private VictorSPX slave2 = new VictorSPX(Ports.Shooter.SLAVE2);
+    private final UnitModel UnitModel = new UnitModel(Constants.Shooter.TICKS_PER_ROTATION);
+
+    /**
+     * this constructor:
+     *  -sets motors inverted
+     *  -makes slaves to follow master
+     *  -sets PID constants

```

Comment:
Don't write this in points write more like sentence and you can generalize your explanation